modules/oe_content_entity/modules/oe_content_entity_organisation/modules/oe_content_organisation_reference/config/install/field.field.oe_contact.oe_organisation_reference.oe_organisation_ref.yml

```diff
@@ -0,0 +1,33 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_contact.oe_organisation_ref
```

Comment:
I'm wondering, maybe we call this reference field `oe_node_reference` as we will use it also in the future to reference the Person content type?

Reply:
If this field will be reused to hold person references within the same solution we have here implemented, I agree to do so.